users/config: Update config article for current config format (v13)

calmh opened the pull request.

AudriusButkevicius commented on users/config.rst:
```diff
@@ -49,8 +49,8 @@ The following shows the default configuration file:
-    <configuration version="12">
-        <folder id="xsdasdSasd" label="default-sync" path="/Users/jb/Sync/" type="readwrite" rescanIntervalS="60" ignorePerms="false" autoNormalize="true">
+    <configuration version="13">
+        <folder id="zj2AA-q55a7" label="Default Folder (zj2AA-q55a7)" path="/Users/jb/Sync/" type="readwrite" rescanIntervalS="60" ignorePerms="false" autoNormalize="true" ro="false">
```
Ro=false?

calmh replied:
That's what we write. Missing omitempty I guess

AudriusButkevicius commented on users/config.rst:
```diff
@@ -98,13 +99,14 @@ The following shows the default configuration file:
-            <databaseBlockCacheMiB>0</databaseBlockCacheMiB>
             <minHomeDiskFreePct>1</minHomeDiskFreePct>
             <releasesURL>https://api.github.com/repos/syncthing/syncthing/releases?per_page=30</releasesURL>
+            <overwriteNames>false</overwriteNames>
```
We should rename this while we can, it's confusing as fuck.

calmh replied:
Bit late in the day for that. Maybe we should delay the v0.13 release for another week and take some time to figure this and the proper solution to the other PR out

AudriusButkevicius replied:
I think this is so low risk that we are fine to do it. Just notify people in the beta page.

AudriusButkevicius replied:
I bet my socks no external code handles this setting yet.

calmh replied:
What do you want to call it? `overwriteRemoteDeviceNamesOnConnect`?

AudriusButkevicius replied:
Yes

calmh commented:
Updated for https://github.com/syncthing/syncthing/pull/3082

AudriusButkevicius commented on users/config.rst:
```diff
@@ -397,22 +418,28 @@ Options Element
-    that the address must have a protocol scheme prefix. Currently ``tcp://``
-    is the only supported protocol scheme.
+    that the address must have a protocol scheme prefix. Currently
+    ``tcp://`` is the only supported protocol scheme. The default value of
```
This is somewhat a lie, we have tcp4/6

calmh replied:
Indeed. I'll fix

rumpelsepp commented:
We have v14 now: https://github.com/syncthing/syncthing/commit/5d2414dfa92cdcbcb55acb8b5d238482af964ec8

calmh commented:
Fixed, extracted listen addresses for sync protocol into a separate section as it's not really a shared syntax with the listen address for the GUI. Needs a little bit more love, but I don't have the time right now :)

AudriusButkevicius commented:
@st-review merge

st-review commented:
:ok_hand: Merged as 0bef7754bb08317dccd78b27635b9b219aab618c. Thanks, @calmh!

AudriusButkevicius commented:
Missing info about default tho
